com.unity.ml-agents/Runtime/AccessUtilities.cs

```diff
@@ -0,0 +1,29 @@
+using UnityEngine;
+
+namespace MLAgents
+{
+    public class AccessUtilities
+    {
+        public static bool CanUpdateModelProperties()
+        {
+            return !Application.isPlaying;
+        }
+
+        public static void LogUnableToUpdate(string className, string propertyName)
+        {
+            Debug.LogWarningFormat("Unable to update {0}.{1} now, as it would affect the NN model parameters.", className, propertyName);
```

Comment:
I would log a very generic message for now : `Cannot edit BehaviorParameters at this time`.

Reply:
Done